CHANGELOG.md

```diff
@@ -300,6 +301,12 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
     `example/cumulus-tf/sync_granule_workflow.asl.json` to include `ACL`
     parameter.
 
+## [v11.1.5] 2022-08-10 [BACKPORT]
+**Please note** changes in 11.1.4 may not yet be released in future versions, as
+this is a backport and patch release on the 11.1.x series of releases. Updates that
+are included in the future will have a corresponding CHANGELOG entry in future
+releases.
+
```

Comment:
I think this is missing the actual changelog entry? https://github.com/nasa/cumulus/pull/3077/files#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4edR15-R22

Reply:
oooohhhhhhh my b

Reply:
there we go